circle.yml

```diff
@@ -0,0 +1,31 @@
+## Customize the test machine
```

Comment:
You can remove this from your PR! I pushed a config file to master https://github.com/julianguyen/ifme/commit/8c7c429244b331eb5af1b276c1c01e7f23cd9d27